### PR TITLE
feat: Add Annotate UDF 

### PR DESCRIPTION
**Pull request by ashmitaraju**
This PR

- [ ] Adds an Annotate UDF which can be used to annotate frames with a bounding box. 
- [ ] Modified the Object Detection tutorial to show how annotate can be used on the output of the YoloV5 UDF directly. 

TODO:

- [ ] Add support to annotate with a label 
- [ ] Add unit test case 
- [ ] Modify the other tutorial notebooks

### CODE REVIEW

File: eva/udfs/ndarray/annotate.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+import numpy as np
+import pandas as pd
+
+from eva.udfs.abstract.abstract_udf import AbstractUDF
+
+color = (207, 248, 64)
+thickness = 4
+
+
+class Annotate(AbstractUDF):
```
Comment (gaurav274) [inline eva/udfs/ndarray/annotate.py:25]: Please use `decorators` to write this UDF.  Also, it is unclear what are the inputs and outputs of the UDF.

File: eva/udfs/ndarray/fuzzy_join.py [hunk 2]
```diff
@@ -31,7 +31,7 @@ def forward(self, df: pd.DataFrame) -> pd.DataFrame:
         Find the distance between two dataframes using <> distance metric.
 
         Returns:
-            ret (pd.DataFrame): The cropped frame.
+            ret (pd.DataFrame): DF containing the distance.
```
Comment (gaurav274) [inline eva/udfs/ndarray/fuzzy_join.py:34]: Also modify it to use `decorators`

**Review by gaurav274 — COMMENTED** [review]
@IshSiva Please provide ideas to transform these udfs into decorators. Thanks!

**Comment by jarulraj** [comment]
@ashmitaraju @IshSiva Can we update this UDF based on `decorators`?

**Comment by jarulraj** [comment]
This is subsumed by #732

**Pull request closed without merging**